regexes/device/mobiles.yml

```diff
@@ -2508,7 +2770,7 @@ Dicam:
 
 # Digi
 Digi:
-  regex: 'Digi ([^);/]+)[);/ ]'
+  regex: 'Digi[ _]([^/;]+)( Build|\))'
```

Comment:
Should we disallow the model name to contain `)` as it was before? Otherwise it might maybe match to much stuff if there are more than one `)` in the useragent :thinking:

Reply:
A fix was needed for
```
Mozilla/5.0 (Linux; Android 8.1.0; Digi_K1) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/75.0.3770.101 Mobile Safari/537.36
```
I now see what could be done better `Digi[ _]([^/;)]+)`

PS Some browsers are beginning to send no UserAgent ")" 
This could be a problem in the future

Reply:
fix

Reply:
Wouldn't it have been enough to add the `_` then and use `Digi[ _]([^);/]+)[);/ ]`? Or didn't match that properly?

Reply:
You're right, thx